src/Mixer/Mixer.cpp

```diff
@@ -0,0 +1,109 @@
+#include "NAS2D/Mixer/Mixer.h"
+
+namespace NAS2D
+{
+
+	Mixer::Mixer(const std::string& name)
+	: mName(name)
+	{
+	}
+
+	Mixer::Mixer()
+	{
+	}
+
+	Mixer::~Mixer()
+	{
+	}
+
+	void Mixer::playSound(Sound& /*sound*/)
+	{
+	}
+
+	void Mixer::stopSound()
+	{
+	}

```

Comment:
If the function was declared pure virtual (`= 0`), no empty function definition would be needed to satisfy the compiler.